x-pack/plugins/security_solution/server/lib/sourcerer/readme.md

```diff
@@ -0,0 +1,78 @@
+# Sourcerer API
+
+### Model reference
+
+```typescript
+interface KibanaDataView {
+  /** Uniquely identifies a Kibana Data View */
+  id: string;
+  /**  list of active patterns that return data  */
+  patternList: string[];
+  /**
+   * title of Kibana Data View
+   * title also serves as "all pattern list", including inactive
```

Comment:
If a user uses the api to edit their data view, does the name/title also automatically update? Trying to understand if this is 100% always a reliable source for now.

Reply:
it updates when they visit the security solution app again. a diff check is run to see if the pattern has updated. that is what step #6 describes below on L45